hyperspy/misc/material.py

```diff
@@ -179,8 +181,7 @@ def _density_of_mixture_of_pure_elements(weight_percent,
     Examples
     --------
     Calculate the density of modern bronze given its weight percent:
-    >>> hs.material.density_of_mixture_of_pure_elements(
-            (88, 12),("Cu", "Sn"))
+    >>> hs.material.density_of_mixture((88, 12),("Cu", "Sn"))
```

Comment:
Docstring: Use list instead of tuples and use single quotation marks to be consistent with other documentation. i.e. [88,12], ['Cu', 'Sn'].
(Same as below)